src/SFA.DAS.Reservations.Application/RefreshCourses/Services/ApprenticeshipCoursesService.cs

```diff
@@ -9,40 +9,22 @@ namespace SFA.DAS.Reservations.Application.RefreshCourses.Services
 {
     public class ApprenticeshipCoursesService : IApprenticeshipCourseService
     {
-        private readonly IFrameworkApiClient _frameworkApiClient;
         private readonly IStandardApiClient _standardApiClient;
 
-        public ApprenticeshipCoursesService(IFrameworkApiClient frameworkApiClient, IStandardApiClient standardApiClient)
+        public ApprenticeshipCoursesService(IStandardApiClient standardApiClient)
         {
-            _frameworkApiClient = frameworkApiClient;
             _standardApiClient = standardApiClient;
         }
 
         public List<Course> GetCourseInformation()
         {
             var list = new ConcurrentBag<Course>();
 
-            var tasks = new List<Task>
-            {
-                GetFrameworks(list),
-                GetStandards(list)
-            };
-
-            Task.WaitAll(tasks.ToArray());
+            GetStandards(list);
```

Comment:
Should be 
```suggestion
var tasks = new List<Task>
            {
                GetStandards(list)
            };

            Task.WaitAll(tasks.ToArray());
```
Still need to await the items returned